global.R

```diff
@@ -81,6 +81,18 @@ conf <- jsonlite::fromJSON("config.json")
 source("helpers.R") # maybe package these or put them in a namespace called "ms"
 source("function_aliases.R")
 
+# blocked watershed char vars
+ms_vars_blocked <<- c(
```

Comment:
@westonslaughter `<<-` is for global assignment. if you're not in a function body, default assignment (`<-`, `=`) is global.